plugins/lsp/lsp-packages.nix

```diff
@@ -243,7 +243,6 @@
     emmet_ls = "emmet-ls";
     emmylua_ls = "emmylua-ls";
     erg_language_server = "erg";
-    erlangls = "erlang-ls";
```

Comment:
It is fine to just move the lsp to the "unpackaged" section when we bump our flake.lock.